runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4194,6 +4195,38 @@ class INTERPRETER_CLASS
 
 		return rc;
 	}
+
+	VMINLINE VM_BytecodeAction
+	inlUnsafeIsFieldAtOffsetFlattened(REGISTER_ARGS_LIST)
+	{
+		I_64 offset = *(I_64*)_sp;
+		j9object_t clz = *(j9object_t*)(_sp + 2);
+
+		I_32 isFlattened = (I_32)FALSE;
+
+		if (NULL != clz) {
+			J9Class *clzJ9Class = J9VM_J9CLASS_FROM_HEAPCLASS(_currentThread, clz);
+			if ((!J9CLASS_IS_ARRAY(clzJ9Class)) && (NULL != clzJ9Class->flattenedClassCache)) {
+
+				// caller provided offset will include header size but entry->offset does not
```

Comment:
From https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-comments, comments should use the /* */ style, rather than // in the native code.

Reply:
changed in 7159341